src/sql/src/plan/statement/ddl.rs

```diff
@@ -2344,7 +2344,8 @@ fn key_constraint_err(desc: &RelationDesc, user_keys: &[ColumnName]) -> PlanErro
 generate_extracted_config!(
     CsrConfigOption,
     (AvroKeyFullname, String),
-    (AvroValueFullname, String)
+    (AvroValueFullname, String),
+    (NullDefaults, bool)
```

Comment:
If you want to make this a little simpler (i.e. avoid the unwrap in purification), you can default the value to `false`. It will still be implied as `true` if the user specifies the option.

```suggestion
    (NullDefaults, bool, Default(false))
```

Reply:
Oh I see, let me do that.